techniques/failures/F1.html

```diff
@@ -117,19 +117,24 @@
       <section class="procedure"><h3>Procedure</h3>
          <p>For content which uses CSS for positioning: </p>
          <ol>
-            <li>Remove the style information from the document or turn off use of
-                                    style sheets in the user agent.</li>
-            <li>Check that the reading order of the content is correct and the
-                                    meaning of the content is preserved.</li>
+            <li>Identify the content elements that use CSS for positioning.</li>
+            <li>Check that the reading order of the content is correct and the meaning of the content is preserved in relation to the surrounding page or context, using one of the following methods:
+                <ul>
+                  <li>Code inspection: review the HTML code to determine the logical reading sequence.</li>
+                  <li>Accessibility tree inspection: examine the accessibility tree to confirm the reading order.</li>
+                  <li>Assistive Technology test: use a screen reader to check how the content is read aloud.</li>
+                  <li>Visual test without CSS positioning: remove positioning styles and observe the visual reading sequence.</li>        
+                </ul>
+            </li>
+           
          </ol>
       </section>
       <section class="results"><h3>Expected Results</h3>
          <ul>
-            <li>If check #2 is false, then this failure condition applies and the
-                                    content fails this Success Criterion.</li>
+           <li>If any options in check #2 is false, then this failure condition applies and the content fails the Success Criterion.</li>
```

Comment:
Minor grammatical error - "If any options in check #2 is false" should either be "If any option in check #2 is false" or "If any options in check #2 are false"